index.html

```diff
@@ -145,5 +145,20 @@ <h2 class="skillTitle">Skills</h2>
       </div>
     </div>
   </section>
+
+  <section class="contact">
+    <div class="contact-me">
+      <h2 class="contactTitle">Contact me</h2>
+      <p class="contactParra">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+      <form class="contactForm">
+        <ul class="contactUl">
+          <li><input class="contactInput" type="text" name="name" placeholder="Full name" maxlength="30" required autocomplete="off"></li>
+          <li><input class="contactInput" type="email" name="email" placeholder="Email" required></li>
+          <li><textarea class="contactArea" name="msg" rows="8" placeholder="Write your message here" maxlength="500" required></textarea></li>
+        </ul>
+        <button class="contactButton" type="submit">Get It Touch</button>
+      </form>

```

Comment:
- [ ] According to the project requirement, you should use [formspree service](https://formspree.io/html/) to Collect data. please implement it. for reference. you should use:
``action` value should be similar to `https://formspree.io/f/{form_id}`
``method` value equals POST`